src/highlighting.c

```diff
@@ -577,7 +577,12 @@ static void styleset_common_init(GKeyFile *config, GKeyFile *config_home)
 	get_keyfile_int(config, config_home, "styling", "line_height",
 		0, 0, &common_style_set.styling[GCS_LINE_HEIGHT]);
 
+	if (&common_style_set.wordchars)
```

Comment:
this test looks off, you don't want to check the address do you?  and there's not need to protect g_free() against NULL either.